kafka/signed_blinded_token_issuer_handler.go

```diff
@@ -147,15 +154,28 @@ OUTER:
 
 				marshaledDLEQProof, err := DLEQProof.MarshalText()
 				if err != nil {
-					return fmt.Errorf("request %s: could not marshal dleq proof: %w", blindedTokenRequestSet.Request_id, err)
+					message := fmt.Sprintf("request %s: could not marshal dleq proof: %s", blindedTokenRequestSet.Request_id, err)
+					temporary, backoff := utils.ErrorIsTemporary(err, &logger)
+					return &utils.ProcessingError{
```

Comment:
what's the reason we're avoiding using `ProcessingErrorFromErrorWithMessage` here?